addons/popochiu/editor/main_dock/popochiu_row/object_row/character_row/popochiu_character_row.gd

```diff
@@ -0,0 +1,75 @@
+@tool
+extends "res://addons/popochiu/editor/main_dock/popochiu_row/object_row/popochiu_object_row.gd"
+
+enum CharacterOptions {
+	DELETE = MenuOptions.DELETE,
+	ADD_TO_CORE = Options.ADD_TO_CORE,
+	CREATE_STATE_SCRIPT = Options.CREATE_STATE_SCRIPT,
+	SET_AS_PC,
+}
+
+const TAG_ICON = preload("res://addons/popochiu/icons/player_character.png")
+const STATE_TEMPLATE = "res://addons/popochiu/engine/templates/character_state_template.gd"
+
+var is_pc := false : set = set_is_pc
+
+#region Godot ######################################################################################
+func _ready() -> void:
+	super()
+	
+	# Assign icons
+	tag.texture = TAG_ICON
+	
+	menu_popup.set_item_disabled(menu_popup.get_item_index(Options.ADD_TO_CORE), true)
+	menu_popup.set_item_disabled(menu_popup.get_item_index(Options.CREATE_STATE_SCRIPT), true)
+
+
+#endregion
+
+#region Virtual ####################################################################################
+func _get_state_template() -> Script:
+	return load(STATE_TEMPLATE)
+
+
+#endregion
+
+#region SetGet #####################################################################################
+func set_is_pc(value: bool) -> void:
+	is_pc = value
+	tag.visible = value
+	
+	menu_popup.set_item_disabled(menu_popup.get_item_index(CharacterOptions.SET_AS_PC), value)
```

Comment:
I see you are emitting a `pc_changed` signal only when the PC is changed by the interface (see line 60).

I think it should be done at this level to be reflected even if the PC change is done programmatically.

Reply:
I'm not sure about this one. What would be the purpose of seeing that in the dock interface while the game is running?

Reply:
Done. Forget my previous comment, I'm stupid.